src/Controllers/HelloWorldController.cs

```diff
@@ -16,7 +16,7 @@ namespace MlNetCore.Controllers
     public class HelloWorldController : Controller
     {
         private readonly IHtmlLocalizer<HelloWorldController> _localizer;
-        private readonly IUnitOfWork _unitOfWork;
+        private IUnitOfWork _unitOfWork { get; }
```

Comment:
change this for
`private readonly IUnitOfWork _unitOfWork;`

Reply:
and add this method after the Constructor method:
`private IUnitOfWork UnitOfWork
        {
            get { return this._unitOfWork; }
        }`

Reply:
this is the convention :P